### DISCUSSION TITLE
Non-cropped thumbnails

### DISCUSSION THREAD

**Discussion by amarshall** [ente-io/ente#3471] [Enhancements]
All thumbnails are currently square and crop the photo. For some content with very wide or tall aspect ratios, this can hide the majority of the image, making it more difficult to browse. Would like to be able to view with thumbnails that shows the entire content, either by non-constant thumbnail containers (i.e. they stretch to fit), or letterboxing.

**Comment by cschreib** [comment]
I have the same request. When sharing an album to others, it is important that photos are shown as they were taken, including the thumbnails. Cropping can ruin a carefully chosen composition, or hide a key part of the picture. There should be an option to disable it.
